src/server/queries/diffs.js

```diff
@@ -13,11 +13,7 @@ var diffs = module.exports = promise.promisifyAll({
     // no return value
     var deleteStr = "DELETE FROM diffs WHERE users_dashboards_signature_hash='" + signatureHash + "';";
     pool.query(deleteStr, function (err, results) {
-      if (err) {
-        callback(err, null);
-      } else {
-        callback(null, "Diffs deleted");
-      }
+      callback(err, "Diffs deleted");
     });
   },
   addAll: function (signatureHash, diffsArray, callback) {
```

Comment:
There's an old typo in this function at line 28. At the end where it's currently:
 + signatureHash + ");";
It should be:
+ signatureHash + "');";

(missing ' just before the close paren)

After fixing this, the post to /api/commits successfully inserts into DB, but the request hangs:
![screen shot 2016-03-28 at 12 05 15 pm](https://cloud.githubusercontent.com/assets/2690580/14087288/0238493a-f4de-11e5-8d09-9e7fc7bc018b.png)